mslice/plotting/pyplot.py

```diff
@@ -2572,15 +2590,18 @@ def imshow(X, cmap=None, norm=None, aspect=None, interpolation=None, alpha=None,
     if hold is not None:
         ax.hold(hold)
     try:
+        if float('.'.join(matplotlib.__version__.split('.')[:2])) >= 1.5:
```

Comment:
Same comment as above. Can we move the version check and only do it once on import of the module?